Content.Goobstation.Shared/Religion/Nullrod/AltarSourceComponent.cs

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 TheBorzoiMustConsume <197824988+TheBorzoiMustConsume@users.noreply.github.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Goobstation.Shared.Religion.Nullrod
+{
+    [RegisterComponent]
+    public sealed partial class AltarSourceComponent : Component
+    {
+
+        /// <summary>
+        /// Which prototype to create.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("rodProto")]
```

Comment:
```suggestion
        [DataField]
```
completely unneccesary [ViewVariables(VVAccess.ReadWrite)] is automatically inherited by DataField.
The field being named RodProto also makes it turn into rodProto automatically in yml so "rodProto" isn't needed either